src/test/javascript/portal/details/NcWmsPanelSpec.js

```diff
@@ -75,12 +76,14 @@ describe('Portal.details.NcWmsPanel', function() {
             ncwmsPanel._addTemporalControls();
             ncwmsPanel.startDateTimePicker.fireEvent('change', ncwmsPanel.startDateTimePicker);
             expect(ncwmsPanel._onDateSelected).toHaveBeenCalled();
+            expect(window.trackUsage).toHaveBeenCalled();
```

Comment:
These tests should check that the correct values are passed in (I think there are 4 here where we don't).